packages/runtime/container-runtime/src/gc/gcHelpers.ts

```diff
@@ -202,3 +256,195 @@ export async function getSnapshotDataFromOldSnapshotFormat(
 		? undefined
 		: { gcState, tombstones: undefined, deletedNodes: undefined };
 }
+
+/**
+ * Concatenates the given GC states and returns the concatenated GC state.
+ */
+export function concatGarbageCollectionStates(
```

Comment:
I expected to see a bunch of Unit Tests for this come over too, do they exist?

Reply:
There are tests for the `runGarbageCollection` method. I forgot to copy them. The others are not unit tested. I will create an issue to add unit tests for them.

Reply:
https://dev.azure.com/fluidframework/internal/_workitems/edit/3825